src/pages/AboutPage.tsx

```diff
@@ -1,8 +1,78 @@
-import React from "react";
+import React, { useState } from "react";
+import { Button } from "@material-ui/core";
 import { Header } from "components/Header";
+import { labels } from "labels";
+
+interface AboutLinkProps {
+  url: string;
+  label: string;
+}
+
+const AboutLink: React.FC<AboutLinkProps> = ({ url, label }) => {
+  return (
+    <div>
+      <a
```

Comment:
I think MUI's `Link` component is better and provides a more readable prop `underline="none"` instead of `style={{ textDecoration: "none" }}`